includes/class-wsuwp-vals-custom-roles.php

```diff
@@ -673,4 +675,42 @@ public function add_certification_date( $user_id, $role ) {
 			update_user_meta( $user_id, 'certification', sanitize_text_field( date( 'Y-m-d' ) ) );
 		}
 	}
+
+	/**
+	 * Allow site Administrators to edit users with a VALS role.
+	 *
+	 * @param array  $caps    The user's actual capabilities.
+	 * @param string $cap     Capability name.
+	 * @param int    $user_id The user ID.
+	 * @param array  $args    Adds the context to the cap (typically the object ID).
+	 *
+	 * @return
+	 */
+	public function admin_edit_vals_roles( $caps, $cap, $user_id, $args ) {
+		$user = get_userdata( $user_id );
```

Comment:
It might make sense to start this off with the `'edit_user' === $cap` check so that the rest of this processing isn't done on some of the other meta cap mapping.